omnigibson/prims/entity_prim.py

```diff
@@ -806,6 +806,10 @@ def update_handles(self):
                 joint.initialize()
             joint.update_handles()
 
+        # Update the physics view if we're articulated
+        if self.articulated:
+            self._physics_view = og.sim.physics_sim_view.create_articulation_view(self.articulation_root_path)
+
```

Comment:
Seems unrelated - remove?

Reply:
Oh sorry, yeah it's unrelated but it's a necessary hotfix. Mass matrix stuff breaks without this line